huxley/core/models.py

```diff
@@ -356,11 +356,11 @@ class Meta:
 
 class Delegate(models.Model):
     school        = models.ForeignKey(School, related_name='delegates', null=True)
-    assignment    = models.ForeignKey(Assignment, related_name='delegates', blank=True)
+    assignment    = models.ForeignKey(Assignment, related_name='delegates', blank=True, null=True)
     name          = models.CharField(max_length=64)
-    email         = models.EmailField(blank=True)
+    email         = models.EmailField(blank=True, null=True)
```

Comment:
Is there a reason why you have `null=True` here? I don't think it's necessary for this field to not be required.